rotkehlchen/chain/ethereum/modules/sushiswap/accountant.py

```diff
@@ -0,0 +1,27 @@
+from typing import TYPE_CHECKING, Dict
+
+from rotkehlchen.accounting.structures.base import get_tx_event_type_identifier
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.accounting.interfaces import ModuleAccountantInterface
+from rotkehlchen.chain.ethereum.accounting.structures import TxEventSettings, TxMultitakeTreatment
+
+from ..constants import CPT_SUSHISWAP_V2
```

Comment:
I think relative import here looks terrible. It would have been nice if they were in the same directory